rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMRest.java

```diff
@@ -31,13 +31,7 @@
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.security.KeyException;
-import java.util.AbstractMap;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
@aminelouati could you please import only the map class from java.util instead of importing all classes? thx

Reply:
Indeed, the IDE replaced automatically 
import java.util.Collection;	
import java.util.Collections;	
import java.util.Iterator;	
import java.util.List;	
import java.util.Map;	
import java.util.Set;

by import java.util.*;

It is no less efficient to use a wildcard import -- all those extra classes will only be loaded if they are used.